File: app/src/application/models/Item.ts
export default class Item
{
    id: number;
    name: string;
    quantity: number;
    price: number;
    value: number;

    constructor(
        id: number,
        name: string,
        quantity: number,
        price: number,
        value: number
    )
    {
        this.id = id;
        this.name = name;
        this.quantity = quantity;
        this.price = price;
        this.value = value;
    }
}

File: server/Application/Models/Item.cs
namespace Ubss.Application.Models;

public sealed class Item
{
    public string Id { get; set; }

    public string Name { get; set; }

    public int Quantity { get; set; }

    public long Price { get; set; }

    public long Value { get; set; }

    public Dictionary<string, string> Properties { get; set; }
}
